src/app/static/src/app/store/filteredData/getters.ts

```diff
@@ -104,26 +108,47 @@ export const getters = {
         for (const region in result) {
             const indicators = result[region];
             if (indicators.art) {
-                indicators.art.color = getColor(indicators.art, artRange, getters.colorFunctions.art);
+                indicators.art.color = getColor(indicators.art!.value, artRange.min, artRange.max, getters.colorFunctions.art);
             }
             if (indicators.prev) {
-                indicators.prev.color = getColor(indicators.prev, prevRange, getters.colorFunctions.prev);
+                indicators.prev.color = getColor(indicators.prev!.value, prevRange.min, prevRange.max, getters.colorFunctions.prev);
             }
         }
         return {
             indicators: result,
             artRange: artRange,
             prevRange: prevRange
         };
+    },
+    flattenedRegionOptions: function(state: FilteredDataState, getters: any, rootState: RootState, rootGetters: any) {
```

Comment:
Added these functions as it's useful be able to just look up a region rather than go hunting for in in a nested tree structure every time.